src/libraries/Microsoft.PowerFx.Core/Public/Config/PowerFxConfig.cs

```diff
@@ -40,6 +40,20 @@ public PowerFxConfig(CultureInfo cultureInfo = null)
             EnumStore = new EnumStore();
         }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="PowerFxConfig"/> class.
+        /// Stopgap constructor until Enum Store is refactored. Do not rely on, this will be removed. 
+        /// </summary>
+        internal PowerFxConfig(CultureInfo cultureInfo = null, EnumStore enumStore = null)
```

Comment:
>PowerFxConfig [](http://example.com/codeflow?start=17&length=13)

nit: this can lead to ambiguous ctor overloads. 
The challenge is we want a public ctor and EnumStore is internal. 

If this is stopgap, then we could we use a a static builder method to avoid the ambiguity?

Reply:
Yeah, good point.